lib/finch/response.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Finch.Response do
+  alias __MODULE__
```

Comment:
maybe since this module is not private, we should add a brief `@moduledoc`?

Reply:
Great job with these docs!

Reply:
Thank you! They are heavily cribbed from the existing `Mint` docs